concent_api/core/models.py

```diff
@@ -625,41 +627,37 @@ class RecipientType(ChoiceEnum):
         Provider    = 'provider'
         Requestor   = 'requestor'
 
-    payment_ts              = DateTimeField()
-    task_owner_key          = BinaryField()
-    provider_eth_account    = CharField(max_length = ETHEREUM_ADDRESS_LENGTH)
+    payment_ts = DateTimeField()
+    task_owner_key = BinaryField(
+        validators=[
+            MinLengthValidator(TASK_OWNER_KEY_LENGTH),
+            MaxLengthValidator(TASK_OWNER_KEY_LENGTH),
+        ]
+    )
+    provider_eth_account = CharField(
+        max_length=ETHEREUM_ADDRESS_LENGTH,
+        validators=[
+            MinLengthValidator(ETHEREUM_ADDRESS_LENGTH),
+            MaxLengthValidator(ETHEREUM_ADDRESS_LENGTH),
```

Comment:
I think this is redundant to `max_length=ETHEREUM_ADDRESS_LENGTH`. Correct me if I'm wrong, but my understanding was that using `max_length=ETHEREUM_ADDRESS_LENGTH` makes Django use `MaxLengthValidator` under the hood.

Reply:
Yes, I was thinking about this and just wanted to make it more visible, but you are right, after thinking it through again I agree it should be removed.

Reply:
Would it be good place for one line comment with this information? @dybi ?

Reply:
:+1: done